src/constants/labels.json

```diff
@@ -0,0 +1,73 @@
+{
+  "overview": {
+    "welcome": "Willkommen bei Lunes!\nLerne Vokabeln für deinen Beruf.",
+    "unit": "Lektion",
+    "units": "Lektionen",
+    "word": "Wort",
+    "words": "Wörter",
+    "exercises": "Übungen"
+  },
+  "header": {
```

Comment:
I'd just move those to overview/home/... from above as well, see comment below.

Reply:
done